src/mame/drivers/gigatron.cpp

```diff
@@ -24,8 +27,14 @@ class gigatron_state : public driver_device
 	}
 
 	void gigatron(machine_config &config);
+	
+	void init_gigatron();
```

Comment:
This is a prototype for an undefined function which isn't even necessary at the moment.